src/mocks/handlers.js

```diff
@@ -0,0 +1,21 @@
+import { rest } from 'msw'
+
+export const handlers = [
+  rest.get(process.env.NASA_APOD_URL, (req, res, ctx) => {
+    req.url.searchParams.set('api_key', process.env.NASA_API_KEY)
```

Comment:
the env var for the API key could be hardcoded and can be anything, I think that having it as a env var is not required for the test

Reply:
😅